src/components/Search/index.js

```diff
@@ -1,16 +1,88 @@
-import React from 'react';
-import { Input } from '@material-ui/core';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
+import { fade, makeStyles } from '@material-ui/core/styles';
+import SearchIcon from '@material-ui/icons/Search';
+import InputBase from '@material-ui/core/InputBase';
+
+const useStyles = makeStyles(theme => ({
+  search: {
+    position: 'relative',
+    borderRadius: theme.shape.borderRadius,
+    backgroundColor: fade(theme.palette.common.white, 0.15),
+    '&:hover': {
+      backgroundColor: fade(theme.palette.common.white, 0.25),
+    },
+    marginLeft: 0,
+    width: '100%',
+    [theme.breakpoints.up('sm')]: {
+      marginLeft: 0,
+      width: 'auto',
+    },
+  },
+  searchIcon: {
+    padding: theme.spacing(0, 2),
+    height: '100%',
+    position: 'absolute',
+    pointerEvents: 'none',
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+  inputRoot: {
+    color: 'inherit',
+  },
+  inputInput: {
+    border: '1px solid #a0a0a0',
+    padding: theme.spacing(1, 1, 1, 0),
+    // vertical padding + font size from searchIcon
+    paddingLeft: `calc(1em + ${theme.spacing(4)}px)`,
```

Comment:
We're switching between px, em, and ch here. It's best to pick one

Reply:
Shortfalls of pasting CSS directly from Material UI (https://material-ui.com/components/app-bar/) 😂 

Will keep in for now since I played around with it and couldn't get it to look right with the `paddingLeft` if I change the dimensions (e.g. `calc(1em + ${theme.spacing(4)}px)` to something simple like `3em)

Reply:
Tidying this up deserves it's own ticket @lpatmo